src/main/java/org/apache/commons/lang3/math/NumberUtils.java

```diff
@@ -1239,6 +1260,7 @@ public static byte max(final byte... array) {
         validateArray(array);
 
         // Finds and returns max
+        @SuppressWarnings("array.access.unsafe.high.constant") // validateArray(array) => array.length != 0
```

Comment:
There are an impressive number of these warning suppressions. We might want to add an `@EnsuresMinLenIf` annotation just to handle them.